src/main/java/org/opensearch/sdk/ssl/SSLConfigConstants.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.sdk.ssl;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.opensearch.common.settings.Settings;
+
+/**
+ * Class containing constants for SSL
+ */
+public final class SSLConfigConstants {
```

Comment:
I see that this and the keystore file contain a lot of functionality that is unused.  What do you think about removing unused functionality or at least making as much as possible package internal?  This might be a good candidate for a follow up issue too.

Reply:
Most of the values in this class are used, but there are a lot of commented out Ciphers which is the same in the security plugin. The list of Ciphers used should always be matching between the 2 systems which underscores the need to extract these into a [library](https://github.com/opensearch-project/opensearch-sdk-java/issues/714). 

In `DefaultSslKeyStore` I commented out the function to `getOtherName` because it would have required the addition of a new dependency on the SDK and I was trying to avoid that. See https://github.com/opensearch-project/opensearch-sdk-java/pull/619#discussion_r1156034710.

I chose to keep the commented out code in to spark a conversation on here to see if its potentially something needed. This was added into the security plugin to support hot reload with certificates that contained `otherName`.